src/Data/Avro.hs

```diff
@@ -81,6 +81,7 @@ module Data.Avro
   , encode
   , encodeContainer
   , encodeContainerWithSync
+  , schemaOf
```

Comment:
It is a very useful function which can be used by the client code.
I use it pretty much everywhere when it comes to working with `Avro` encodable data types.